### PR TITLE
Ltd 4117 appeal documents

### PR DESCRIPTION
**Pull request by kevincarrogan**
### Aim

Adds an endpoint to create appeal documents.

[LTD-4117](https://uktrade.atlassian.net/browse/LTD-4117)

### CODE REVIEW

File: api/appeals/views.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+from django.http import Http404
+
+from api.core.authentication import ExporterAuthentication
+
+from .models import Appeal
+from .serializers import AppealDocumentSerializer
+
+
+class AppealDocuments(CreateAPIView):
```
Comment (currycoder) [inline api/appeals/views.py:11]: So just to understand, exporters upload some set of supporting documents for their appeal.  Is that right?

Comment (currycoder) [inline api/appeals/views.py:11]: Might we also want to list the documents in the future?  I guess for now this is keeping the scope limited.

**Review by seijihg — APPROVED** [review]

**Review by currycoder — APPROVED** [review]

File: api/appeals/views.py (hunk 1 above)
Comment (kevincarrogan) [reply api/appeals/views.py:11]: That is correct. They will upload zero or more supporting documents for their appeal.

Comment (kevincarrogan) [reply api/appeals/views.py:11]: So far I've found I don't need to create a separate list endpoint but adding the appeal object to an existing endpoint seems to do the job.

**Pull request merged**